server.go

```diff
@@ -150,6 +155,14 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 		ret.EventWorker.Work()
 	}()
 
+	ret.TraceWorker = NewTraceWorker(ret.statsd)
+	go func() {
+		defer func() {
+			ret.ConsumePanic(recover())
+		}()
+		ret.TraceWorker.Work()
+	}()
```

Comment:
do we need to run the traceworker if we have no traceaddress? wondering if this should be moved into the `if len(conf.TraceAddress) > 0` conditional